docs/sphinx/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openmicroscopy</groupId>
+    <artifactId>ome-model</artifactId>
+    <version>5.3.2-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+
+  <groupId>org.openmicroscopy</groupId>
+  <version>5.3.2-SNAPSHOT</version>
+
+  <artifactId>ome-model-docs</artifactId>
```

Comment:
Shall I update this PR accordingly?  Which other components need tweaking, bioformats?